dojo/tools/veracode_sca/parser.py

```diff
@@ -166,9 +175,19 @@ def get_findings_csv(self, file, test):
 
             severity = self.fix_severity(row.get("Severity", None))
             cvss_score = float(row.get("CVSS score", 0))
-            date = datetime.strptime(
-                row.get("Issue opened: Scan date"), "%d %b %Y %H:%M%p %Z"
-            )
+            # Get the date based on the first_seen setting
+            try:
+                if settings.USE_FIRST_SEEN:
+                    date = datetime.strptime(
+                        row.get("Issue opened: Scan date"), "%d %b %Y %H:%M%p %Z"
+                    )
+                else:
+                    date = datetime.strptime(
+                        row.get("Issue opened: Scan date"), "%d %b %Y %H:%M%p %Z"
```

Comment:
These as well

Reply:
They are the same, but this time it is intentional. In the unit test reports for this parser, there is not any available data in the report to satisfy the notion of a "last seen" date. Before this change, the `Issue opened: Scan date` field was used, so it seemed appropriate to maintain that same behavior for both cases.

I am open to any suggestions that could make more sense though!

Reply:
same as https://github.com/DefectDojo/django-DefectDojo/pull/9361/files#r1459718238